processor/abci.go

```diff
@@ -1169,8 +1169,13 @@ func (app *App) DeliverPropose(ctx context.Context, tx abci.Tx, deterministicId
 		// TODO(): for now we are using a hash of the market ID to create
 		// the lp provision ID (well it's still deterministic...)
 		lpid := hex.EncodeToString(vgcrypto.Hash([]byte(nm.Market().ID)))
-		err := app.exec.SubmitMarketWithLiquidityProvision(
-			ctx, nm.Market(), nm.LiquidityProvisionSubmission(), party, lpid, deterministicId)
+		var err error
+		if lp := nm.LiquidityProvisionSubmission(); lp != nil {
+			err = app.exec.SubmitMarketWithLiquidityProvision(
+				ctx, nm.Market(), nm.LiquidityProvisionSubmission(), party, lpid, deterministicId)
+		} else {
+			err = app.exec.SubmitMarket(ctx, nm.Market(), party)
```

Comment:
Calling this will call `_ = mkt.StartOpeningAuction(ctx)` which puts the market in a pending state. When the market proposal gets enacted we'll try to start the opening auction again, and fail because: 
```
if m.mkt.State != types.MarketStateProposed {
		return ErrCannotStartOpeningAuctionForMarketNotInProposedState
	}
```